phpBB/styles/prosilver/theme/common.css

```diff
@@ -603,6 +603,93 @@ a.header-avatar span:after {
 	display: none !important;
 }
 
+/* Smiley Dropdown---------------------------------------- */
+
+.dropdown .smiley-content {
+	border: 1px solid transparent;
+	border-radius: 5px;
+	position: relative;
+	z-index: 2;
+	display: inline-block;
+	overflow: hidden;
+	overflow-y: auto;
+	width: 200px;
+	max-height: 300px;
+	padding: 5px;
+}
+
+.smiley-box ul {
+	display: inline;
+	margin: 0;
+}
+
+.smiley-box .smiley-list {
+	border: none;
+	display: inline-block;
+}
+
+.smiley-box .smiley-list a {
+	text-decoration: none;
+	padding: 0;
+}
+
+.dropdown-right .smiley-content {
+	right: 20px;
+}
+
+.dropdown-up .smiley-content {
+	bottom: -4px;
+}
+
+.button .dropdown-left .dropdown {
+	right: -12px;
+}
+
+.button .dropdown-right .dropdown {
+	left: -12px;
+}
+
+.dropdown-left .smiley-content {
+	left: 20px;
+}
+
+.button .dropdown .pointer {
+	top: 0;
+}
+
```

Comment:
its not required to be centered look at the pointer on the topics page for the tools menu on area51 its not centered!
![screenshot 2017-04-23 18 14 22](https://cloud.githubusercontent.com/assets/73081/25317915/b5f23cbe-2850-11e7-8d79-fa36b6e9cda6.png)